kornia/contrib/extract_patches.py

```diff
@@ -125,12 +106,13 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:  # type: ignore
 
 
 def extract_tensor_patches(
-        input: torch.Tensor,
-        window_size: Union[int, Tuple[int, int]],
-        stride: Optional[Union[int, Tuple[int, int]]] = 1,
-        padding: Optional[Union[int, Tuple[int, int]]] = 0) -> torch.Tensor:
+    input: torch.Tensor,
+    window_size: Union[int, Tuple[int, int]],
+    stride: Optional[Union[int, Tuple[int, int]]] = 1,
```

Comment:
this is an inherited mistake from our side, but `Optional` should be only used for possible `None` values. We could fix now